src/site/content/en/media/video-and-source-tags/index.md

```diff
@@ -0,0 +1,336 @@
+---
+layout: post
+title: The <video> and <source> tags
+authors:
+  - samdutton
+  - joemedley
+description: |
+  You've properly prepared a video file for the web. You've given it correct
+  dimensions and the correct resolution. You've encrypted it. You've even
+  created separate webm and mp4 files for different browsers. For anyone to see
+  it, you still need to add it to a web page.
+date: 2014-14-15
+updated: 2020-08-20
+---
```

Comment:
@jpmedley I plan on redirecting the old H5R guide (https://www.html5rocks.com/en/tutorials/video/basics/) to this page (`/video-and-source-tags`) once this PR lands. Can you check the old H5R guide and see if there are any useful/interesting ideas that should be incorporated into this new web.dev page?

Reply:
I'm incorporating this one too. Take it off your list. https://www.html5rocks.com/en/tutorials/track/basics/